### ISSUE TITLE
I need help

### ISSUE THREAD

**Issue by Frax-Python** [UnamSanctam/UnamWebPanel#231]
i executed the miner on my pc and the uninstaller has been deleted pelase help me guys

**Comment by Frax-Python** [comment]
i cant stop the process, please help me bro

**Comment by Frax-Python** [comment]
i need to reinstall windows i think, because i cant stop the virus

**Comment by UnamSanctam** [comment]
You don't need to, remove the startup entry in either the Task Scheduler or the registry and then immediately restart your computer. That's the easiest way to do it.

**Comment by Frax-Python** [comment]
i need to stop the process console host on task manager and immediately restart my computer?

**Comment by UnamSanctam** [comment]
No, if you ran it as administrator then open up the Task Scheduler, press the first folder on the left and then right-click the entry called `GoogleUpdateTaskMachineQC` and press "Remove". Then restart your computer.

Or you can start CMD as administrator and then run `schtasks /delete /tn GoogleUpdateTaskMachineQC /f` and then restart the computer.

**Comment by Frax-Python** [comment]
i did it, i think its still working

**Comment by UnamSanctam** [comment]
Only way would be to check if the process is still running.

**Comment by Frax-Python** [comment]
thanks bro, i dont have the virus anymore

**Comment by UnamSanctam** [comment]
Alright great.
